packages/builders/src/node/build/schema.json

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
Did you copy this from the webpackbuilder schema file?

Reply:
Mostly, some of the options are new so I wrote those manually